### DISCUSSION TITLE
Unable to change TV select action

### DISCUSSION THREAD

**Discussion by diimaan** [umbrellaplug/umbrellaplug.github.io#143] [Q&A] [answered]
The TV select action option is set to Auto Play and even after changing it to Source Select (Window xml skin) option it reverts back to Auto Play.

**Comment by umbrellaplug** [comment, marked as answer]
Turn OFF Settings->TV Shows->Continuous Episode Playback Settings->Enable PlayNext popup.

**Reply by diimaan** [reply]
Thanks! I missed the help section on Continuous Episode Playback Settings.
